assets/src/setup/pages/save/index.js

```diff
@@ -41,16 +106,84 @@ export function Save() {
 	}, [ didSaveDeveloperToolsOption, savingDeveloperToolsOption, saveDeveloperToolsOption ] );
 
 	if ( savingOptions || savingDeveloperToolsOption ) {
-		return <Loading />;
+		return <Saving />;
 	}
 
-	if ( ! didSaveOptions || ! didSaveDeveloperToolsOption ) {
-		return null;
+	let heading = __( 'Congratulations!', 'amp' );
+	if ( 'standard' === themeSupport ) {
+		heading = __( 'Your site is ready', 'amp' );
 	}
 
 	return (
-		<div>
-			{ __( 'Options saved', 'amp' ) }
+		<div className="done grid grid-6-5">
+			<div>
+				<h1>
+					{ heading }
+				</h1>
+				<p>
+					{ getDescription( themeSupport ) }
+				</p>
+			</div>
+			<div className="done__preview-container">
+				<AMPInfo icon={ ( props ) => <IconMobile { ...props } /> }>
+					{ __( 'Live view of your site', 'amp' ) }
+				</AMPInfo>
+				<Phone>
+					<iframe
+						className="done__preview-iframe"
+						src={ addQueryArgs( PREVIEW_PERMALINK, 'standard' === themeSupport ? {} : { amp: 1 } ) }
+						title={ __( 'Site preview', 'amp' ) }
+						name="amp-wizard-completion-preview"
+					/>
+				</Phone>
+
+				{ 'reader' !== themeSupport && (
+					<Button
+						isPrimary
+						href={
+							addQueryArgs(
+								PREVIEW_PERMALINK,
+								'transitional' === themeSupport
+									? { amp: 1 }
+									: {},
+							)
+						}
```

Comment:
I suppose manipulating the query vars is needed because the output of `amp_admin_get_preview_permalink` may now be stale.

Here's an idea: can the REST API endpoint include the preview link URL? Perhaps a new endpoint? Or part of the options endpoint as a read-only value? Then when saving we will get the most up-to-date preview URL based on the settings. This will be very important when we allow sites to customize how the URLs are constructed for AMP.  See #2204.

Reply:
I actually considered (1) including the preview link in the options response after saving, and (2) creating a new REST endpoint just for the preview link. #1 feels wrong because it's not an option, and #2 felt like overkill for a single field, but with your suggestion I'll go ahead with #1. It is sort of an option, just not one that persists in the DB.